Gemfile

```diff
@@ -33,6 +33,7 @@ group :development, :test do
 
   if Gem.ruby_version >= Gem::Version.new("3.0.0")
     gem "webrick"
+    gem "rackup"
```

Comment:
I don't think there is any reason to include rackup based on Ruby version. Rackup is needed when Rack 3+ is used.